RELEASING.md

```diff
@@ -1,13 +1,73 @@
 # Releasing Ziti
 
-Release happen by merging from the release-next branch to main.
+## Release-next Pre-requisites
 
-## Release pre-requisites
+Perform these steps in PR branches based on release-next (trunk).
 
-1. Make sure you have a clean build in GitHub Actions.
-2. Make sure you have a clean build in Jenkins smoketest (until smoketest is superceded by fablab smoke in GH)
-3. Make sure CHANGELOG.md is up to date
-    1. You can use `ziti-ci build-release-notes` to generate ziti library version updates and issues fixed,
-       as long as the git commit has `fixed #<issue number>` (or fixes, closes, closed, etc)
-4. Create a PR to merge release-next to main, once it's approved you can merge and the release will happen.
+1. Tidy dependencies.
+    1. Ensure you have downloaded the `@latest` artifact from the dependency(ies) you are updating in the main Ziti project, e.g.,
 
+        ```bash
+        go get -u github.com/openziti/edge@latest
+        ```
+
+    2. Run `go mod tidy` in the main Ziti project and in the `./zititest` sub-tree.
+
+        ```bash
+        go mod tidy
+        cd ./zititest
+        go mod tidy
+        cd ..
+        ```
+
```

Comment:
`go test ./...` will also build, so running the `go build ./...` step is unnecessary